doc/Editing-the-Wiki.md

```diff
@@ -100,6 +100,41 @@ You can then add the image in markdown using the URL `test.png`:
 
     ![alternative text](test.png)
 
+## Mermaid Diagrams
+Mermaid diagrams can be added using the following syntax (example):
+```
+```{mermaid}
+---
+title: Simple sample
+---
+stateDiagram-v2
+    [*] --> Document
+    Develop_software --> Document
+    Document --> Develop_software
+    Develop_software --> Test_software
+    Test_software --> Crash
+    Crash --> Develop_software
+    Test_software --> [*]
+```
```

Comment:
There's a trick here for getting that trailing
````
```
````

to render:

`````suggestion
````
```{mermaid}
---
title: Simple sample
---
stateDiagram-v2
    [*] --> Document
    Develop_software --> Document
    Document --> Develop_software
    Develop_software --> Test_software
    Test_software --> Crash
    Crash --> Develop_software
    Test_software --> [*]
```
````
`````